src/main/java/org/mockbukkit/mockbukkit/entity/LivingEntityMock.java

```diff
@@ -52,10 +52,12 @@
 import org.jetbrains.annotations.Nullable;
 import org.jetbrains.annotations.Range;
 import org.mockbukkit.mockbukkit.simulate.entity.LivingEntitySimulation;
+import org.w3c.dom.Attr;
```

Comment:
Might have missed this, but this import is probably not necessary (another nitpick)